Makefile

```diff
@@ -85,25 +85,43 @@ check: ## Concurrently runs a whole bunch of static analysis tools
 .PHONY: oc-generate-deployments
 oc-generate-deployments: $(OC_DEPLOYMENTS) ## Creates openshift deployments for ike-prow plugins
 
-.PHONY: oc-init
-oc-init:
+
+OC_PROJECT_NAME?=ike-prow-plugins
+.PHONY: oc-init-project
+oc-init-project: ## Initiliazes new project with config maps and secrets
 	@echo "Setting cluster project"
-	@oc new-project ike-prow-plugins
+	@oc new-project $(OC_PROJECT_NAME)
 	@oc create configmap plugins && oc create configmap config
-	@oc create secret generic hmac-token --from-file=hmac=config/hmac.token && oc create secret generic oauth-token --from-file=oauth=config/oauth.token
+	@oc create secret generic hmac-token --from-file=hmac=config/hmac.token
+	@oc create secret generic oauth-token --from-file=oauth=config/oauth.token
+	@oc create secret generic sentry-dsn --from-file=sentry=config/sentry.dsn
```

Comment:
in the README, there is no mention about the `sentry.dsn` file. It is only in the documentation: http://arquillian.org/ike-prow-plugins/#_sentry_logging where is written: `Plugins in this repository can be integrated with Sentry` which means that Sentry should be optional. 
When I run it locally on minishift, I guess that Sentry is not required, right? At least it is not mentioned in the prerequisites. But the Makefile fails when it tries to load the `sentry.dsn` file.

Reply:
Yeah, noticed that. Will think how to fix it and also cleanup secret creation as it's very much copy-paste

Reply:
Created new issue #52

Reply:
Brought `Makefile` improvements in https://github.com/arquillian/ike-prow-plugins/pull/53